src/handlers/next/create-app-register-handler.ts

```diff
@@ -16,33 +16,49 @@ const debug = createDebug("createAppRegisterHandler");
 
 type HookCallbackErrorParams = {
   status?: number;
-  body?: object;
   message?: string;
 };
 
 class RegisterCallbackError extends Error {
   public status = 500;
 
-  public body: object = {};
-
   constructor(errorParams: HookCallbackErrorParams) {
     super(errorParams.message);
 
     if (errorParams.status) {
       this.status = errorParams.status;
     }
-
-    if (errorParams.body) {
-      this.body = errorParams.body;
-    }
   }
 }
 
-const createCallbackError = (params: HookCallbackErrorParams) => new RegisterCallbackError(params);
+const createCallbackError = (params: HookCallbackErrorParams) => {
+  throw new RegisterCallbackError(params);
+};
+
+export type RegisterHandlerResponseBody = {
+  success: boolean;
+  error?: {
+    code?: string;
+    message?: string;
+  };
+};
+export const createRegisterHandlerResponseBody = (
```

Comment:
Debug message in this fn could be helpfull. WDYT?

Reply:
Really? It only does simple mapping and this error goes directly to the response, I don't see a benefit really